okapi/services/caches/save_user_coords/docs.xml

```diff
@@ -0,0 +1,30 @@
+<xml>
+    <brief>Update personal coordinates of a geocache</brief>
```

Comment:
I agree. Would it make sense to make `cache_code` actually a "|" separated list of `cache codes` and `user_coords` a matching list (by position) of coordinates, such that only one request needs to be made for potentially a large set of caches? Such scenarios are quite common on some alternate geocaching platforms but I haven't seen a use case on opencaching.de as there are very few powertrails which would benefit from such a functionality.

Alternatively I have thought a about making this an additional, optional parameter to `save_personal_note`, however, in the interest of guaranteeing to not break existing client applications I decided against it.

With respect to your question: I don't think many more such single property changing methods are required. So I'd consider this a one of a kind. On my roadamp there is at this point a method let me call it "upload_fieldnotes", an issue for that exists and it has been discussed for years.

Other than that, if I can afford the time, I would like to add a full circle CRUD functionality for "lists". I can provide more details via a separate issue.

Reply:
We already have methods which [retrieve fields from multiple entities](https://github.com/opencaching/okapi/assets/2168535/311812d5-e081-4304-a550-e59700191be0), and corresponding methods which retrieve fields from a single entity (e.g. geocache, entry, etc.). Since that's the style already, we should follow the same pattern with saving methods.

So, we could have:

- `update_geocache` which would take `cache_code` and fields such as `personal_note`, `user_coords`, `marked_as_watched`, `marked_as_ignored`, `passwd`, etc. (This method would replace `edit`, `mark`, `save_personal_notes` and `save_user_coords`.)
- `update_geocaches` which would take a JSON list of arguments for `update_geocache`, similar to what `search_and_retrieve` does.

> With respect to your question: I don't think many more such single property changing methods are required. So I'd consider this a one of a kind.

It's fine to leave it as is then. We can always clean it up later (in a backward-compatible way).